base/sort.jl

```diff
@@ -1140,6 +1437,18 @@ right(o::Perm) = Perm(right(o.order), o.data)
 lt(::Left, x::T, y::T) where {T<:Floats} = slt_int(y, x)
 lt(::Right, x::T, y::T) where {T<:Floats} = slt_int(x, y)
 
+Serial.serialize(x::Float32, ::Left) = Serial.serialize(reinterpret(Int32, x), Reverse)
```

Comment:
Why `Int32` rather than `UInt32`? Maybe worth a comment?

Reply:
Why is so that elements serialized with Left would compare properly with elements serialized with Right. This is unnecessary, so I've simplified it. It now serializes directly and efficiently to a UInt.